lib/ui/theme/theme.dart

```diff
@@ -48,7 +50,11 @@ class PostsTheme {
       onBackground: Color(0xFF646464),
     );
     final iconTheme = IconThemeData(color: Color(0xFF646464));
-    return from(colorScheme: scheme, iconTheme: iconTheme);
+    final mergeTextTheme = Typography.blackCupertino;
```

Comment:
I'm not a big fan of this to be honest. It would mean forcing iOS-styled fonts inside Android application as well, which is not a good practice. I think keep using Roboto on Android is better.

From the Flutter docs: 

> A material design text theme with dark glyphs based on San Francisco.

This could be replaced with

```suggestion
    final mergeTextTheme = defaultTargetPlatform == TargetPlatform.iOS
          ? Typography.blackCupertino
          : Typography.blackMountainView;
```